meson_options.txt

```diff
@@ -8,6 +8,10 @@ option('packet-socket', type: 'feature', value: 'auto', description : 'Enable pa
 option('tests', type: 'boolean', value: true, description: 'Build tests')
 option('fast-heartbeat', type: 'boolean', value: false, description: 'Enable faster heartbeat rate')
 
+# Parameters
+
+option('gv-buffers', type: 'integer', min: 1, value: 16, description: 'Number of buffers used to receive GVSP packets')
```

Comment:
Please rename the parameter to gv-n-buffers

Reply:
Done